### PR TITLE
Bug specials characters into entity and variable chips

### LINKED ISSUE #26: Bug with brackets into the entity
Context
On a Windows 10 computer, with google chrome installed
The command block, look like an single entity.

What are you trying to achieve or the steps to reproduce ?
Adding a closed bracket into the entity.

Which result you had ?
The closed bracket is written outside of the entity.

capture

TODO
Check if the code of the entities have an exceptions for the closing brackets.

### PR DESCRIPTION
**Pull request by david-talabard**
fix #26

### CODE REVIEW

**Comment by willdurand** [comment]
Since the `build` method is static, we can test it without too much effort. Could you add a test case to cover your changes? In addition:

1. add "Fix " before the issue in your PR description (e.g., `Fix #26`)
2. run `yarn lint --fix` to lint + fix the code before committing it

Thanks!

**Comment by david-talabard** [comment]
yes @willdurand , I add some tests to cover this fix

File: tests/shared/components/actionEditor.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      />
+    );
+    expect(wrapper.state("items").length).toEqual(1);
+  });
+
+  it("renders entity with brackets characters", () => {
+    const items = [
+      { type: "variable", text: "<<variable>>" }
+    ];
+    const content = ActionEditor.build(items);
```
Comment (willdurand) [inline tests/shared/components/actionEditor.test.js]: Why do you need snapshot tests for testing this static method?

Comment (david-talabard) [reply tests/shared/components/actionEditor.test.js]: I keep one test with snapshot and add an other specific test for the build method

**Pull request merged**